pydicom/dataset.py

```diff
@@ -1047,6 +1042,43 @@ def save_as(self, filename, write_like_original=True):
 
         pydicom.dcmwrite(filename, self, write_like_original)
 
+    def ensure_file_meta(self):
+        """Create an empty file meta dataset if none exists."""
+        self.file_meta = getattr(self, 'file_meta', None) or Dataset()
+
+    def fix_meta_info(self, include_checks=True):
+        """Ensure the file meta info exists and has the correct values
+        for transfer syntax and media storage uids.
+
+        .. note::
+
+            The transfer syntax for is_implicit_VR = False and
+            is_little_endian = True is ambiguous and will therefore not set.
+
```

Comment:
Would it be better to have this as a warning instead?

Reply:
Done.